### PR TITLE
Mn 2020 02 add captcha

### PR DESCRIPTION
**Pull request by MagdaN**

### CODE REVIEW

File: adhocracy-plus/assets/js/captcheck.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+window.onload = function () {
```
Comment (rmader) [inline adhocracy-plus/assets/js/captcheck.js]: Is this file mostly copied? Then we should add the license text

Comment (MagdaN) [reply adhocracy-plus/assets/js/captcheck.js]: Yes, I just changed it in a way that the api url is not hard coded and adapted it to our linting rules. Will add the license

**Comment by MagdaN** [comment]
@rmader then I will remove the app from the setting and the registration form (I just added it for testing)

**Comment by rmader** [comment]
Would it be possible to check in the registration form if its activated? Just like we check for the servicekonto app?

**Pull request merged**